### ISSUE TITLE
Eufy C120 homekit source can't be watched

### ISSUE THREAD

**Issue by davidanthoff** [AlexxIT/go2rtc#833]
I run the go2rtc add-on in home assistant. I have a Eufy C120 camera that I have configured as a homekit source in go2rtc and I'm also using the homekit module proxy support to expose the camera to my Apple Home ecosystem. The proxy stuff works, i.e. I can see the camera stream on my iPhone, but I only get a black screen when I try to watch the WebRTC or MSE stream that go2rtc is producing for the camera.

My config looks like this (I replaced a bunch of data with `XXX` for privacy reasons):
```
streams:
  dining_room_camera: 
    - homekit://192.168.1.XX:XXX?XXX
    - ffmpeg:dining_room_camera#audio=aac#audio=opus

homekit:
  dining_room_camera:
    pin: XXX
    name: Dining Room Camera
    pairings:
      - client_id=XXX
      - client_id=XXX  
```
When I open `stream.html` for the camera in one tab (which then just shows black), and then look at `info.json` in another tab I get this:
```
{
    "producers": [
        {
            "type": "HomeKit active producer",
            "url": "homekit://192.168.1.XX:XXXX?XXXX",
            "sdp": "{Codecs:[{CodecType:0 CodecParams:[{ProfileID:[1] Level:[0 2] PacketizationMode:0 CVOEnabled:[0] CVOID:[]}] VideoAttrs:[{Width:1920 Height:1080 Framerate:30} {Width:1280 Height:720 Framerate:30} {Width:640 Height:360 Framerate:30} {Width:480 Height:270 Framerate:30} {Width:320 Height:180 Framerate:30} {Width:1280 Height:960 Framerate:30} {Width:1024 Height:768 Framerate:30} {Width:640 Height:480 Framerate:30} {Width:480 Height:360 Framerate:30} {Width:320 Height:240 Framerate:30}] RTPParams:[]}]}\n{Codecs:[{CodecType:2 CodecParams:[{Channels:1 Bitrate:0 SampleRate:[1] RTPTime:[]}] RTPParams:[] ComfortNoise:[]}] ComfortNoise:0}",
            "medias": [
                "video, recvonly, H264",
                "audio, recvonly, ELD/16000/1",
                "video, recvonly, JPEG"
            ],
            "receivers": [
                "0 H264, bytes=0, senders=2",
                "0 ELD/16000/1, bytes=0, senders=1"
            ]
        },
        {
            "type": "RTSP passive producer",
            "url": "rtsp://127.0.0.1:8554/e95703d871db388b6c9aba5a3604f803",
            "remote_addr": "127.0.0.1:32956",
            "user_agent": "ffmpeg/go2rtc",
            "sdp": "v=0\r\no=- 0 0 IN IP4 127.0.0.1\r\ns=go2rtc/1.8.4\r\nc=IN IP4 127.0.0.1\r\nt=0 0\r\na=tool:libavformat 60.3.100\r\nm=audio 0 RTP/AVP 96\r\nb=AS:69\r\na=rtpmap:96 MPEG4-GENERIC/16000/1\r\na=fmtp:96 profile-level-id=1;mode=AAC-hbr;sizelength=13;indexlength=3;indexdeltalength=3; config=140856E500\r\na=control:streamid=0\r\nm=audio 0 RTP/AVP 97\r\nb=AS:64\r\na=rtpmap:97 opus/48000/2\r\na=control:streamid=1\r\n",
            "medias": [
                "audio, recvonly, MPEG4-GENERIC/16000",
                "audio, recvonly, OPUS/48000/2"
            ],
            "receivers": [
                "96 MPEG4-GENERIC/16000, bytes=0, senders=1",
                "97 OPUS/48000/2, bytes=0, senders=1"
            ]
        }
    ],
    "consumers": [
        {
            "type": "WebRTC/WebSocket async passive consumer",
            "remote_addr": "udp4 prflx 192.168.1.28:60817 related :0",
            "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36 Edg/120.0.0.0",
            "medias": [
                "video, sendonly, VP8, RTX, VP9, H264, AV1, RED, ULPFEC, FLEXFEC-03",
                "audio, sendonly, OPUS/48000/2, RED/48000/2, G722/8000, PCMU/8000, PCMA/8000, CN/8000, TELEPHONE-EVENT/48000, TELEPHONE-EVENT/8000, L16, PCML"
            ],
            "senders": [
                "102 H264, bytes=0, receivers=1",
                "111 OPUS/48000/2, bytes=0, receivers=1"
            ]
        },
        {
            "type": "MSE/WebSocket active consumer",
            "remote_addr": "192.168.1.28, 172.30.32.1, 172.30.32.2:33928",
            "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36 Edg/120.0.0.0",
            "medias": [
                "video, sendonly, H264, H265",
                "audio, sendonly, MPEG4-GENERIC, PCMA, PCMU, L16, PCML, OPUS"
            ],
            "senders": [
                "0 H264, bytes=0, receivers=1",
                "96 MPEG4-GENERIC/16000, bytes=0, receivers=1"
            ]
        },
        {
            "type": "RTSP passive consumer",
            "url": "rtsp://127.0.0.1:8554/dining_room_camera?audio",
            "remote_addr": "127.0.0.1:32968",
            "user_agent": "go2rtc/ffmpeg",
            "sdp": "v=0\r\no=- 1 1 IN IP4 0.0.0.0\r\ns=go2rtc/1.8.4\r\nc=IN IP4 0.0.0.0\r\nt=0 0\r\nm=audio 0 RTP/AVP 96\r\na=rtpmap:96 MPEG4-GENERIC/16000/1\r\na=fmtp:96 streamtype=5;profile-level-id=1;mode=AAC-hbr;sizelength=13;indexlength=3;indexdeltalength=3;config=f8ec3000\r\na=control:trackID=0\r\n",
            "medias": [
                "audio, sendonly, ANY"
            ],
            "senders": [
                "96 ELD/16000/1, bytes=0, receivers=1"
            ]
        }
    ]
}
```

Happy to run more diagnostics if that would help get to the bottom of this.

**Comment by AlexxIT** [comment]
Start from checking errors in logs

**Comment by davidanthoff** [comment]
The log for the go2rtc add-on has this:
```
13:26:31.178 INF go2rtc version 1.8.4 linux/amd64
13:26:31.179 INF [api] listen addr=:1984
13:26:31.179 INF [rtsp] listen addr=:8554
13:26:31.179 INF [webrtc] listen addr=:8555
13:26:59.760 WRN github.com/AlexxIT/go2rtc/internal/streams/producer.go:171 > error=EOF url=ffmpeg:dining_room_camera#audio=aac#audio=opus
```
If there are other logs I should look at or change some log setting, I'd appreciate a pointer on what exactly would be useful.

**Comment by davidanthoff** [comment]
@AlexxIT just wanted to follow up on this: is there anything I could do to help diagnose this further?

**Comment by davidanthoff** [comment]
It seems fixed in 1.8.5.
